wrappers/python/tests/TestOpcWater.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import unittest
+import openmm as mm
+from openmm import app
+from openmm import unit
+
+
+class TestOpcWater(unittest.TestCase):
+
+    def test_OpcEnergy(self):
+        pdb = app.PDBFile('../openmm/app/data/opcbox.pdb')
+        topology, positions = pdb.topology, pdb.positions
+        self.assertEqual(len(positions), 864)
+        forcefield = app.ForceField('opc.xml')
+        system = forcefield.createSystem(
+            topology,
+            nonbondedMethod=app.PME,
+            nonbondedCutoff=0.7*unit.nanometer,
+            constraints=app.HBonds,
+            rigidWater=True,
+        )
+
+        box_vectors = pdb.topology.getPeriodicBoxVectors()
+        system.setDefaultPeriodicBoxVectors(*box_vectors)
```

Comment:
You don't need to do this.  The default box vectors automatically get set from the topology.

Reply:
removed (and tested, it works)